### PR TITLE
A Boring End Rod

### LINKED ISSUE #24: Boring End Rod
End Rod minus the particles

### PR DESCRIPTION
**Pull request by itsbazke**
😴

### CODE REVIEW

File: src/main/java/com/lovetropics/extras/block/BoringEndRodBlock.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+    @Override
+    public void animateTick(BlockState pState, Level pLevel, BlockPos pPos, RandomSource pRandom) {
+        //It's boring, so don't animate!
+    }
+
+    @Override
+    public int getLightEmission(BlockState state, BlockGetter level, BlockPos pos) {
+        return 14; //This is the same as a normal end rod
```
Comment (Gegy) [inline src/main/java/com/lovetropics/extras/block/BoringEndRodBlock.java]: I think these are usually set through the block properties - it should also be possible to ‘steal’ the properties of the normal end rod with the `.initialProperties()` function on `BlockBuilder` :D

Comment (itsbazke) [reply src/main/java/com/lovetropics/extras/block/BoringEndRodBlock.java]: Ooh you are very right! That's what I get for late night coding :D

Comment (itsbazke) [reply src/main/java/com/lovetropics/extras/block/BoringEndRodBlock.java]: what about the model generation stuff, it feels like there is a way to steal all that from the end rod too?

Comment (Gegy) [reply src/main/java/com/lovetropics/extras/block/BoringEndRodBlock.java]: I don't think so - at least not for the blockstate file. But there might be a Registrate helper for generating that kind of blockstate file?

**Review by Gegy — APPROVED** [review]

**Pull request merged**